packages/extension/src/pages/register/connect-keystone/index.tsx

```diff
@@ -0,0 +1,141 @@
+import { observer } from "mobx-react-lite";
+import React, { FunctionComponent, ReactChild } from "react";
+import { useRegisterHeader } from "../components/header";
+import {
+  useSceneEvents,
+  useSceneTransition,
+} from "../../../components/transition";
+import { RegisterSceneBox } from "../components/register-scene-box";
+import { Stack } from "../../../components/stack";
+import { Box } from "../../../components/box";
+import { Button } from "../../../components/button";
+import { ColorPalette } from "../../../styles";
+
+export const ConnectKeystoneScene: FunctionComponent<{
+  name: string;
+  password: string;
+
+  // append mode일 경우 위의 name, password는 안쓰인다. 대충 빈 문자열 넣으면 된다.
+  appendModeInfo?: {
+    vaultId: string;
+    afterEnableChains: string[];
+  };
```

Comment:
@renfengshi I think I'll delete the `appendModeInfo` parameter since it's not used in the current file.